example/node/index.js

```diff
@@ -185,6 +187,25 @@ async function run() {
   const fs = require('fs');
   const imageBlob = await layerS2L2A.getMap(getMapParams, ApiType.WMS);
   fs.writeFileSync('./image.jpeg', imageBlob, { encoding: null });
+
+  // Cancel request example
+  // Create source and pass token as config on the request (getMap WMS/Processing and getStats supported for layerV3)
+  const token = new CancelToken();
+
+  setTimeout(() => {
+    printOut('Cancelling request');
+    token.cancel();
+  }, 100);
+
+  try {
+    const response = await layerS2L2A.getMap(getMapParams, ApiType.WMS, { cancelToken: token });
+    console.log(response);
```

Comment:
Nitpick: since we don't usually output an image to the console, I would suggest:
```
console.log("Image was received in less than 100ms.");
```

Fftr.

Reply:
Type: `recieved` should be `received`. Fftr.